recipes/stompest/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "stompest" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: 83a4d9d9627028953e7bec71a958a60488243f7ea6f41c2a428ad95f533cedce
+  - path: .
```

Comment:
```suggestion
```

Reply:
Thanks for reviewing. The License file is not part of the pip package. I use the `path: .` trick to include one. What is the good way to include it?